### ISSUE TITLE
Changes not applying instantly

### ISSUE THREAD

**Issue by ShayBox** [quidsup/notrack#38]
Its always taken over 24 hours, for changes to apply but it gets annoying when all my local ip's are blocks and i have to whitelist them and wait a day

**Comment by quidsup** [comment]
Theres an "Update Blocklist" button on the white list page, and another button under Options.
The button under Options forces a re-download of all blocklists.

I can't apply them instantly because it will cause denial of service for low powered devices like the RPi 1.

I don't understand why you need to unblock IP's? The purpose of a DNS server is to resolve a name to an IP. 
DNS can't resolve an IP to an IP because its already done.

Its possible a program maybe trying http://www.192.168.1.1, which will fail to resolve
And then it tries http://192.168.1.1 which bypasses the DNS server

**Comment by ShayBox** [comment]
im just having problems with the ip blocking because i have 2 local pc's
hosting web servers (including the pc running the notrack) and i can goto
the notrack website but any other subdirectory that's not /admin will be
blocked

**Comment by quidsup** [comment]
Thats not IP blocking, you need to tell /etc/lighttpd/lighttpd.conf the location of the other web folders.
See line 42 for more more info
https://github.com/quidsup/notrack/blob/master/conf/lighttpd.conf
